modules/ROOT/pages/sid-reference.adoc

```diff
@@ -0,0 +1,210 @@
+= Sensitive Information Detection
+
+== WAF Security Policy Summary Log
+
+When you apply the WAF policy to the Runtime Fabric, a `rtfWafSecurityPolicySummary` log is generated, which shows the WAF category, rule ID, and count for how many times the violation was detected. 
+For example:
+
+[json]
+----
+{
+	"timeStamp": "2019-04-15T16:13:00.045Z",
+	"collectionIntervalInMinutes": "1",
+	"node": "node-host",
+	"wafSummary": {
+		"requestScannerDetection": [
+
+		],
+		"requestProtocolEnforcement": [
+
+		],
+		"requestProtocolAttack": [
+
+		],
+		"requestApplicationAttackLfi": [
+
+		],
+		"requestApplicationAttackRfi": [
+
+		],
+		"requestApplicationAttackRce": [
+
+		],
+		"requestApplicationAttackPhp": [
+
+		],
+		"requestApplicationAttackXss": [
+
+		],
+		"requestApplicationAttackSqli": [
+
+		],
+		"requestApplicationAttackSessionFixation": [
+
+		],
+		"requestSidPiiCore": [{
+			"ruleId": "600100",
+			"count": 1
+		}],
+		"requestSidPiiEnhanced": [
+
+		],
+		"requestSidPci": [
+
+		],
+		"requestSidPhi": [
+
+		],
+		"responseDataLeakages": [
+
+		],
+		"responseDataLeakagesSql": [
+
+		],
+		"responseDataLeakagesJava": [
+
+		],
+		"responseDataLeakagesPhp": [
+
+		],
+		"responseDataLeakagesIis": [
+
+		],
+		"responseSidPiiCore": [
+
+		],
+		"responseSidPiiEnhanced": [
+
+		],
+		"responseSidPci": [
+
+		],
+		"responseSidPhi": [
+
+		]
+	}
+}
+----
+
+== SID Summary Log
+
+When you apply the WAF policy to the Runtime Fabric, a `rtfSidSummary` log is generated, which shows the WAF category, rule ID, and count for how many times the violation was detected. 
```

Comment:
"which shows the ...detected" --> which shows the offending API, rule description, direction, and count for the number of times the sid rule was triggered.